instrumentation/instaawssdk/propagation_test.go

```diff
@@ -86,7 +66,7 @@ func TestSpanContextFromSQSMessage(t *testing.T) {
 	})
 }
 
-func TestSpanContextFromSQSMessage_LegacyHeaders(t *testing.T) {
+func TestSpanContextFromSQSMessage_2(t *testing.T) {
```

Comment:
Hm, good catch.
As far as I can see, there is nothing here related to the legacy headers, and also, this test seems to be already covered in the original `TestSpanContextFromSQSMessage`.
@sanojsubran would you be so kind and double check, just to be sure?
It sounds to me that we can remove this test completely.

Reply:
But this is not related to this PR, so the removal (or not) of this test should not prevent the PR to be approved.

Reply:
Right, I meant to ask you folks about this test when putting the PR up, but then I forgot about that.

AFAICT it really has nothing to do with legacy headers. I tried to check the git history to see if this test has ever been about legacy headers but AFAICT it was just misnamed from the start.

If you both think that it's covered in the other test (I guess it is) then let's just remove the test in this PR. If, on the other hand, it covers some aspect that is not covered in the other test then we should probably find a better name instead of `_2` :-) 

Let's wait for Sanoj's input and then we can decide.

Reply:
I agree with you guys. It seems to me also that we can remove it.

Reply:
done